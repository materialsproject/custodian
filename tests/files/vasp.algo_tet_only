 -----------------------------------------------------------------------------
|                                                                             |
|           W    W    AA    RRRRR   N    N  II  N    N   GGGG   !!!           |
|           W    W   A  A   R    R  NN   N  II  NN   N  G    G  !!!           |
|           W    W  A    A  R    R  N N  N  II  N N  N  G       !!!           |
|           W WW W  AAAAAA  RRRRR   N  N N  II  N  N N  G  GGG   !            |
|           WW  WW  A    A  R   R   N   NN  II  N   NN  G    G                |
|           W    W  A    A  R    R  N    N  II  N    N   GGGG   !!!           |
|                                                                             |
|      ALGO=A and IALGO=5X tend to fail with the tetrahedron method           |
|      (e.g. Bloechls method ISMEAR=-5 is not variational)                    |
|      please switch to IMSEAR=0-n, except for DOS calculations               |
|      For DOS calculations use IALGO=53 after preconverging with ISMEAR>=0   |
|          I HOPE YOU KNOW, WHAT YOU ARE  DOING                               |
|                                                                             |
 -----------------------------------------------------------------------------
