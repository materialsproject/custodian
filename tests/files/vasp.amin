 running on    4 total cores
 distrk:  each k-point on    1 cores,    4 groups
 distr:  one band on    1 cores,    1 groups
 OpenACC runtime initialized ...    4 GPUs detected
 vasp.6.3.2 27Jun22 (build Jun 29 2023 14:50:22) complex                         
 POSCAR found type information on POSCAR CdI 
 POSCAR found :  2 types and      99 ions
 Reading from existing POTCAR
 scaLAPACK will be used selectively (only on CPU)
 Reading from existing POTCAR
 LDA part: xc-table for Pade appr. of Perdew
 -----------------------------------------------------------------------------
|                                                                             |
|           W    W    AA    RRRRR   N    N  II  N    N   GGGG   !!!           |
|           W    W   A  A   R    R  NN   N  II  NN   N  G    G  !!!           |
|           W    W  A    A  R    R  N N  N  II  N N  N  G       !!!           |
|           W WW W  AAAAAA  RRRRR   N  N N  II  N  N N  G  GGG   !            |
|           WW  WW  A    A  R   R   N   NN  II  N   NN  G    G                |
|           W    W  A    A  R    R  N    N  II  N    N   GGGG   !!!           |
|                                                                             |
|     The requested file  could not be found or opened for reading            |
|     k-point information. Automatic k-point generation is used as a          |
|     fallback, which may lead to unwanted results.                           |
|                                                                             |
 -----------------------------------------------------------------------------

 -----------------------------------------------------------------------------
|                                                                             |
|           W    W    AA    RRRRR   N    N  II  N    N   GGGG   !!!           |
|           W    W   A  A   R    R  NN   N  II  NN   N  G    G  !!!           |
|           W    W  A    A  R    R  N N  N  II  N N  N  G       !!!           |
|           W WW W  AAAAAA  RRRRR   N  N N  II  N  N N  G  GGG   !            |
|           WW  WW  A    A  R   R   N   NN  II  N   NN  G    G                |
|           W    W  A    A  R    R  N    N  II  N    N   GGGG   !!!           |
|                                                                             |
|     One of the lattice vectors is very long (>50 A), but AMIN is rather     |
|     large. This can spoil convergence since charge sloshing might occur     |
|     along the long lattice vector. If problems with convergence are         |
|     observed, try to decrease AMIN to a smaller value (e.g. 0.01).          |
|     Note: This warning only applies if the self-consistency cycle is        |
|     used.                                                                   |
|                                                                             |
 -----------------------------------------------------------------------------

 INCAR ok, starting setup
 POSCAR ok, starting setup
 FFT: planning ... GRIDC
 FFT: planning ... GRID_SOFT
 FFT: planning ... GRID
 WAVECAR not read
 entering main loop
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1     0.654864287549E+04    0.65486E+04   -0.42351E+05 21824   0.168E+03
DAV:   2     0.386898823739E+03   -0.61617E+04   -0.59196E+04 25120   0.328E+02
DAV:   3    -0.172568026248E+03   -0.55947E+03   -0.55436E+03 25024   0.932E+01
DAV:   4    -0.186061501271E+03   -0.13493E+02   -0.13459E+02 24880   0.139E+01
DAV:   5    -0.186461852896E+03   -0.40035E+00   -0.40026E+00 27216   0.240E+00    0.439E+01
RMM:   6    -0.162644121003E+03    0.23818E+02   -0.52634E+01 21774   0.820E+00    0.327E+01
