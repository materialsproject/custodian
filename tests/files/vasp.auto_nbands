---

| |
| W W AA RRRRR N N II N N GGGG !!! |
| W W A A R R NN N II NN N G G !!! |
| W W A A R R N N N II N N N G !!! |
| W WW W AAAAAA RRRRR N N N II N N N G GGG ! |
| WW WW A A R R N NN II N NN G G |
| W W A A R R N N II N N GGGG !!! |
| |
| The number of bands has been changed from the values supplied in |
| the INCAR file. This is a result of running the parallel version. |
| The orbitals not found in the WAVECAR file will be initialized with |
| random numbers, which is usually adequate. For correlated |
| calculations, however, you should redo the groundstate calculation. |
| I found NBANDS = 4. Now, NBANDS = 64. |
| |

---
