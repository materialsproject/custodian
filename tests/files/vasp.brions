 ----------------------------------------------------
    OOO  PPPP  EEEEE N   N M   M PPPP
   O   O P   P E     NN  N MM MM P   P
   O   O PPPP  EEEEE N N N M M M PPPP   -- VERSION
   O   O P     E     N  NN M   M P
    OOO  P     EEEEE N   N M   M P
 ----------------------------------------------------
 running  128 mpi-ranks, with    1 threads/rank
 distrk:  each k-point on    8 cores,   16 groups
 distr:  one band on    4 cores,    2 groups
 vasp.6.2.1 16May21 (build Jul 28 2021 23:24:15) complex                        
  
 POSCAR found type information on POSCAR NaPd
 POSCAR found :  2 types and      48 ions
 scaLAPACK will be used
 LDA part: xc-table for Pade appr. of Perdew
 POSCAR, INCAR and KPOINTS ok, starting setup
 FFT: planning ...
 WAVECAR not read
 WARNING: random wavefunctions but no delay for mixing, default for NELMDL
 entering main loop
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1     0.259055125689E+04    0.25906E+04   -0.16206E+05 14616   0.116E+03
DAV:   2     0.103107001085E+03   -0.24874E+04   -0.23892E+04 12824   0.366E+02
DAV:   3    -0.105189937336E+03   -0.20830E+03   -0.19766E+03 13224   0.111E+02
DAV:   4    -0.112580098164E+03   -0.73902E+01   -0.73565E+01 13296   0.209E+01
DAV:   5    -0.112904391648E+03   -0.32429E+00   -0.32425E+00 18952   0.354E+00    0.839E+00
RMM:   6    -0.148517521639E+03   -0.35613E+02   -0.98053E+01 14356   0.746E+00    0.439E+01
RMM:   7    -0.110332207642E+03    0.38185E+02   -0.12318E+01 12659   0.489E+00    0.844E+00
RMM:   8    -0.106664042444E+03    0.36682E+01   -0.48493E+00 11998   0.293E+00    0.598E+00
RMM:   9    -0.104643696045E+03    0.20203E+01   -0.10147E+00 13543   0.131E+00    0.514E+00
RMM:  10    -0.104048278353E+03    0.59542E+00   -0.14678E+00 14422   0.169E+00    0.346E+00
RMM:  11    -0.102964854905E+03    0.10834E+01   -0.87664E-01 13106   0.140E+00    0.115E+00
RMM:  12    -0.102940032088E+03    0.24823E-01   -0.18891E-01 13019   0.701E-01    0.700E-01
RMM:  13    -0.102940241863E+03   -0.20977E-03   -0.42376E-02 14938   0.338E-01    0.380E-01
RMM:  14    -0.102931719898E+03    0.85220E-02   -0.13292E-02 14208   0.221E-01    0.222E-01
RMM:  15    -0.102933391371E+03   -0.16715E-02   -0.84351E-03 13773   0.178E-01    0.190E-01
RMM:  16    -0.102935183535E+03   -0.17922E-02   -0.28242E-03 14846   0.877E-02    0.210E-01
RMM:  17    -0.102932600362E+03    0.25832E-02   -0.15400E-03 15212   0.526E-02    0.118E-01
RMM:  18    -0.102932044171E+03    0.55619E-03   -0.10001E-03 13062   0.406E-02    0.570E-02
RMM:  19    -0.102931980507E+03    0.63664E-04   -0.46192E-04 12448   0.226E-02    0.370E-02
RMM:  20    -0.102931972342E+03    0.81648E-05   -0.18480E-04 11436   0.139E-02    0.307E-02
RMM:  21    -0.102931888343E+03    0.84000E-04   -0.10503E-04  9862   0.115E-02    0.211E-02
RMM:  22    -0.102931827616E+03    0.60727E-04   -0.65112E-05  8545   0.896E-03    0.125E-02
RMM:  23    -0.102931813487E+03    0.14129E-04   -0.40516E-05  7547   0.700E-03    0.688E-03
RMM:  24    -0.102931811977E+03    0.15096E-05   -0.23161E-05  6823   0.598E-03
   1 F= -.10293181E+03 E0= -.10292786E+03  d E =-.102932E+03  mag=     0.0000
 BRION: g(F)=  0.345E-01 g(S)=  0.000E+00
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.102899636845E+03    0.32177E-01   -0.15620E+01 12832   0.438E+00    0.704E-01
RMM:   2    -0.102937053172E+03   -0.37416E-01   -0.43446E-01 11856   0.837E-01    0.364E-01
RMM:   3    -0.103023627522E+03   -0.86574E-01   -0.27301E-01 12782   0.496E-01    0.514E-01
RMM:   4    -0.102965331214E+03    0.58296E-01   -0.21515E-02 12452   0.229E-01    0.308E-01
RMM:   5    -0.102965004682E+03    0.32653E-03   -0.22573E-03 13291   0.797E-02    0.274E-01
RMM:   6    -0.102964462169E+03    0.54251E-03   -0.30661E-04 12986   0.323E-02    0.238E-01
RMM:   7    -0.102963772008E+03    0.69016E-03   -0.38430E-04 11823   0.236E-02    0.159E-01
RMM:   8    -0.102963224529E+03    0.54748E-03   -0.13659E-03 11440   0.361E-02    0.183E-02
RMM:   9    -0.102963096009E+03    0.12852E-03   -0.13331E-04  9410   0.150E-02    0.340E-02
RMM:  10    -0.102963055487E+03    0.40521E-04   -0.28043E-05  7189   0.912E-03    0.213E-02
RMM:  11    -0.102963041225E+03    0.14263E-04   -0.22262E-05  6251   0.628E-03    0.737E-03
RMM:  12    -0.102963024193E+03    0.17031E-04   -0.90867E-06  5465   0.483E-03    0.263E-03
RMM:  13    -0.102962996773E+03    0.27420E-04   -0.32324E-06  4933   0.361E-03    0.933E-04
RMM:  14    -0.102962996980E+03   -0.20679E-06   -0.18719E-06  4247   0.272E-03
   2 F= -.10296300E+03 E0= -.10295913E+03  d E =-.311850E-01  mag=    -0.0000
 BRION: g(F)=  0.266E-01 g(S)=  0.000E+00 retain N=  1 mean eig= 7.96
 eig:   7.961
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.994701690334E+02    0.34928E+01   -0.10021E+03 14840   0.468E+01    0.342E+00
RMM:   2    -0.102111364016E+03   -0.26412E+01   -0.27072E+01 12555   0.796E+00    0.197E+00
RMM:   3    -0.105012737756E+03   -0.29014E+01   -0.98692E+00 13058   0.451E+00    0.278E+00
RMM:   4    -0.103108466143E+03    0.19043E+01   -0.88538E-01 12291   0.176E+00    0.152E+00
RMM:   5    -0.103100276279E+03    0.81899E-02   -0.17322E-01 14285   0.757E-01    0.124E+00
RMM:   6    -0.103088428079E+03    0.11848E-01   -0.19343E-02 16431   0.236E-01    0.112E+00
RMM:   7    -0.103080627639E+03    0.78004E-02   -0.58199E-03 15923   0.113E-01    0.943E-01
RMM:   8    -0.103068433445E+03    0.12194E-01   -0.23762E-02 14068   0.189E-01    0.334E-01
RMM:   9    -0.103067013361E+03    0.14201E-02   -0.23575E-02 15078   0.180E-01    0.340E-01
RMM:  10    -0.103066647687E+03    0.36567E-03   -0.29334E-03 13086   0.974E-02    0.189E-01
RMM:  11    -0.103065852547E+03    0.79514E-03   -0.15767E-03 13955   0.559E-02    0.943E-02
RMM:  12    -0.103066142116E+03   -0.28957E-03   -0.77897E-04 13176   0.318E-02    0.437E-02
RMM:  13    -0.103065725374E+03    0.41674E-03   -0.44892E-04 12673   0.269E-02    0.243E-02
RMM:  14    -0.103065601189E+03    0.12419E-03   -0.17993E-04 10001   0.149E-02    0.822E-03
RMM:  15    -0.103065628085E+03   -0.26896E-04   -0.78806E-05  7456   0.886E-03    0.900E-03
RMM:  16    -0.103065589527E+03    0.38558E-04   -0.56294E-05  6575   0.826E-03    0.165E-03
RMM:  17    -0.103065600434E+03   -0.10906E-04   -0.40249E-05  5705   0.599E-03    0.167E-03
RMM:  18    -0.103065605072E+03   -0.46380E-05   -0.22501E-05  5362   0.488E-03
   3 F= -.10306561E+03 E0= -.10306259E+03  d E =-.102608E+00  mag=    -0.0000
 BRION: g(F)=  0.893E-03 g(S)=  0.000E+00 retain N=  1 mean eig= 7.58
 eig:   7.581
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.103051852801E+03    0.13748E-01   -0.36420E+00 13088   0.202E+00    0.331E-01
RMM:   2    -0.103061313695E+03   -0.94609E-02   -0.91641E-02 11308   0.375E-01    0.191E-01
RMM:   3    -0.103075014448E+03   -0.13701E-01   -0.70317E-02 12535   0.238E-01    0.314E-01
RMM:   4    -0.103067963942E+03    0.70505E-02   -0.40583E-03 12272   0.116E-01    0.172E-01
RMM:   5    -0.103066994283E+03    0.96966E-03   -0.48514E-04 11156   0.408E-02    0.145E-01
RMM:   6    -0.103066880284E+03    0.11400E-03   -0.12565E-04 11776   0.184E-02    0.120E-01
RMM:   7    -0.103066861635E+03    0.18649E-04   -0.54305E-05  7021   0.104E-02    0.956E-02
RMM:   8    -0.103066835740E+03    0.25895E-04   -0.16637E-04  8255   0.131E-02    0.460E-02
RMM:   9    -0.103066882138E+03   -0.46398E-04   -0.99208E-05  7372   0.105E-02    0.740E-03
RMM:  10    -0.103066935738E+03   -0.53600E-04   -0.19741E-05  6020   0.620E-03    0.849E-03
RMM:  11    -0.103066922194E+03    0.13544E-04   -0.38862E-06  5234   0.411E-03    0.784E-03
RMM:  12    -0.103066922609E+03   -0.41535E-06   -0.30265E-06  4769   0.313E-03
   4 F= -.10306692E+03 E0= -.10306385E+03  d E =-.131754E-02  mag=    -0.0000
 BRION: g(F)=  0.593E-03 g(S)=  0.000E+00 retain N=  1 mean eig= 5.69
 eig:   5.691
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.103066226425E+03    0.69577E-03   -0.31317E-01 12712   0.619E-01    0.967E-02
RMM:   2    -0.103067217849E+03   -0.99142E-03   -0.80205E-03  9944   0.116E-01    0.548E-02
RMM:   3    -0.103066772148E+03    0.44570E-03   -0.60800E-03 10588   0.684E-02    0.632E-02
RMM:   4    -0.103068094344E+03   -0.13222E-02   -0.22655E-04  9764   0.272E-02    0.534E-02
RMM:   5    -0.103067543842E+03    0.55050E-03   -0.31373E-05  8208   0.105E-02    0.434E-02
RMM:   6    -0.103067545144E+03   -0.13020E-05   -0.88685E-06  5782   0.570E-03    0.362E-02
RMM:   7    -0.103067541195E+03    0.39490E-05   -0.29841E-06  4914   0.370E-03    0.340E-02
RMM:   8    -0.103067534653E+03    0.65423E-05   -0.19398E-06  4649   0.314E-03
   5 F= -.10306753E+03 E0= -.10306447E+03  d E =-.612044E-03  mag=    -0.0000
 BRION: g(F)=  0.530E-03 g(S)=  0.000E+00
 BRIONS problems: POTIM should be increased
 retain N=  1 mean eig=11.24
 eig:  11.242
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.102942393423E+03    0.12515E+00   -0.31507E+01 12736   0.632E+00    0.956E-01
RMM:   2    -0.103018338176E+03   -0.75945E-01   -0.82319E-01 12492   0.118E+00    0.546E-01
RMM:   3    -0.103091385214E+03   -0.73047E-01   -0.60869E-01 12856   0.705E-01    0.602E-01
RMM:   4    -0.103091203211E+03    0.18200E-03   -0.23542E-02 12444   0.272E-01    0.536E-01
