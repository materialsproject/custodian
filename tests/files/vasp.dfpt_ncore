 -----------------------------------------------------------------------------
|                                                                             |
|     EEEEEEE  RRRRRR   RRRRRR   OOOOOOO  RRRRRR      ###     ###     ###     |
|     E        R     R  R     R  O     O  R     R     ###     ###     ###     |
|     E        R     R  R     R  O     O  R     R     ###     ###     ###     |
|     EEEEE    RRRRRR   RRRRRR   O     O  RRRRRR       #       #       #      |
|     E        R   R    R   R    O     O  R   R                               |
|     E        R    R   R    R   O     O  R    R      ###     ###     ###     |
|     EEEEEEE  R     R  R     R  OOOOOOO  R     R     ###     ###     ###     |
|                                                                             |
|     The PEAD routines do not work for NCORE /= 1 (or NPAR /= NCPU).         |
|     Several features internally rely on the PEAD routines, so even if       |
|     you have not specified LPEAD=.TRUE., try restarting your job with       |
|     NCORE=1.                                                                |
|                                                                             |
|       ---->  I REFUSE TO CONTINUE WITH THIS SICK JOB ... BYE!!! <----       |
|                                                                             |
 -----------------------------------------------------------------------------
