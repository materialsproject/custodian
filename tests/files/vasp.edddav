Error EDDDAV: Call to ZHEGV failed. Returncode =   7 1   8