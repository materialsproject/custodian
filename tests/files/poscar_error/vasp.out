Found some non-integer element in rotation matrix
